test/cppsuite/test_harness/test.h

```diff
@@ -127,6 +131,7 @@ class test {
         _thread_manager->join();
 
         /* Validation stage. */
+        testutil_check(_configuration->get_bool(ENABLE_TRACKING, enable_tracking));
```

Comment:
Call is_enabled from the tracking component instead of pulling out the configuration again.

Reply:
Add is_enabled to the component class seeing as we're using it in a number of places now.